shared/ViewFormatters/CourseExtensions.cs

```diff
@@ -10,6 +10,11 @@ namespace GovUk.Education.SearchAndCompare.UI.Shared.ViewFormatters
 {
     public static class CourseExtensions
     {
+        public const string FundingOption_Salary = "Salary";
+        public const string FundingOption_Scholarship = "Scholarship, bursary or student finance if you’re eligible";
+        public const string FundingOption_Bursary = "Bursary or student finance if you’re eligible";
+        public const string FundingOption_StudentFinance = "Student finance if you’re eligible";
+
         private static Regex websiteRegex = new Regex("^https?:\\/\\/");
```

Comment:
Is it bad that I haven't done Regexes for so long that my brain parsed `"https?:\\/\\/"` as a malformed ascii emoji?